src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/HdcLicence.kt

```diff
@@ -103,6 +103,13 @@ data class HdcLicence(
   @JsonFormat(pattern = "dd/MM/yyyy")
   override val licenceExpiryDate: LocalDate? = null,
 
+  @Schema(
+    description = "The offender’s actual home detention curfew date",
+    example = "06/05/2023",
+  )
+  @JsonFormat(pattern = "dd/MM/yyyy")
+  val homeDetentionCurfewActualDate: LocalDate? = null,
```

Comment:
**question:**  does this need to be set somewhere?

Reply:
Is `homeDetentionCurfewActualDate` not being set in the `createHdc` function in the licence factory?

Reply:
Added `homeDetentionCurfewActualDate` to `ToModelTransformers.kt`